c2corg_ui/static/js/map/diffmap.js

```diff
@@ -81,10 +80,13 @@ app.DiffMapController = function(mapFeatureCollection) {
   app.utils.setupSmartScroll(mouseWheelZoomInteraction);
 
   if (!mapFeatureCollection) {
-    this.map.setView(new ol.View({
-      center: app.MapController.DEFAULT_CENTER,
-      zoom: app.MapController.DEFAULT_ZOOM
-    }));
+    var mapSize = this.map.getSize();
```

Comment:
That would indeed be more consistent with the changes in the map directive.
I just wonder if we should not do it directly in the ``ol.Map`` object creation instead?

By the way do we really need to display a map if there is no ``mapFeatureCollection`` in the diff page? What about skipping this whole block since the diff template only use the diffmap directive when geometries are available: https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/templates/document/diff.html#L52-L66 ?